Makefile

```diff
@@ -0,0 +1,130 @@
+# SPDX-License-Identifier: CC0-1.0
+#
+# SPDX-FileContributor: Antonio Niño Díaz, 2023
+
+# Source code paths
+# -----------------
+
+SOURCEDIRS	:= src
+INCLUDEDIRS	:= src
+
+# Libraries
+# ---------
+
+LIBS		:= -lm
+LIBDIRS		:=
+
+# Build artifacts
+# ---------------
+
+NAME		:= ptexconv
+BUILDDIR	:= build
+ELF		:= $(NAME)
+
+# Tools
+# -----
+
+STRIP		:= -s
+BINMODE		:= 755
+
+CC		:= gcc
+CXX		:= g++
+CP		:= cp
+MKDIR		:= mkdir
+RM		:= rm -rf
+MAKE		:= make
+INSTALL		:= install
+
+# Verbose flag
+# ------------
+
+ifeq ($(VERBOSE),1)
+V		:=
+else
+V		:= @
+endif
+
+# Source files
+# ------------
+
+SOURCES_C	:= $(shell find -L $(SOURCEDIRS) -name "*.c")
+SOURCES_CPP	:= $(shell find -L $(SOURCEDIRS) -name "*.cpp")
```

Comment:
Also, I'm not sure if it makes sense to keep the C++ rules here. Do you prefer it if I remove them?